db/migrate/20160322195653_move_switch_host_to_jointable.rb

```diff
@@ -0,0 +1,25 @@
+class MoveSwitchHostToJointable < ActiveRecord::Migration[5.0]
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    has_many :switches,
+             :class_name => 'MoveSwitchHostToJointable::Switch'
```

Comment:
Personally, I would one-line these has_many lines and ignore rubocop but that's just me.  I find the single line more readable in this situation.